RELEASE.md

```diff
@@ -64,177 +72,303 @@ A release for the bpfman project is comprised of the following major components:
-a PR. This must go through the regular PR review process and get merged into the
-`main` branch. Approval of the PR indicates community consensus for a new
-release.
+1. Go to https://github.com/bpfman/bpfman/releases.
+2. Push the "Draft a new release" button.
+3. Enter the new release number in the "Choose a tag" pull-down.
+4. Choose the most recent release in the "Previous tag" pull-down.
+5. Push the "Generate release notes" button.
+
+The automatically generated output will likely need to be reorganized and
+cleaned up a bit, but it provides a good starting point.
+
+### Standard Release from Main Branch
+
+This section describes the standard release process used when making a release
+from the main branch and may be used for major, minor, or patch releases.  As
+mentioned above, we first complete the release for `bpfman` and then follow that
+up with a release for `bpfman-operator`.
+
+#### bpfman Release
+
+- Create a new branch in your `bpfman` fork named, for example
+  `<githubuser>/release-x.y.z`, and use the new branch in the upcoming steps.
+- Make the following changes
+    - Add a new changelog for the release using the process described in
+      [Generating Release Notes](#generating-release-notes).
+    - Update the
+      [Cargo.toml](https://github.com/bpfman/bpfman/blob/main/Cargo.toml)
+      version for the workspace:
+        - `version = "x.y.z"`
+        - `bpfman = { version = "x.y.z", path = "./bpfman" }"`
+        - `bpfman-api = { version = "x.y.z", path = "./bpfman-api" }`
+        - Note: `bpfman-csi` does not need to be updated.
+    - Run `cargo generate-lockfile`
+    - Add a new `examples` config directory for the release version.
+    - Modify the tag for `XDP_DISPATCHER_IMAGE` and `TC_DISPATCHER_IMAGE` in
+      `bpfman/src/lib.rs` from `latest` to the new release tag.
+- Commit the changes, push them to your repo, and open a PR against the `bpfman`
+  repo.
+- After the PR is reviewed, merged, and all GitHub actions have completed
+  successfully, tag the release with the version number (e.g., v0.5.2).
+  - Tag the release using the commit on `main` where the changelog update
+    merged.
+  - A maintainer or someone with write permission on the repo must create the
+    tag.
+  - This can be done using the `git` CLI or Github's [release][release] page.
+- The Release will be automatically created by GitHub actions when the tag is
+  applied.
+
+After these steps are completed, the following should occur:
+- All [GitHub actions][gh-actions] should complete successfully.
+- The release appears on the GitHub Releases Page.
+- Images are built and updated with the new version tag at:
+    - quay.io/bpfman
+    - quay.io/bpfman-bytecode
+    - quay.io/bpfman-userspace
+- The new version appears at [crates.io](https://crates.io/crates/bpfman)
+- A new RPM is built and pushed to the [bpfman COPR
+  repository](https://copr.fedorainfracloud.org/coprs/g/ebpf-sig/bpfman/).
```

Comment:
we also we need to undo the changes in bpfman/src/lib.rs and pointed back to latest right ? or revert the PR